src/onboarding/containers/LoginPage.tsx

```diff
@@ -21,9 +20,6 @@ export const LoginPage: FC = () => (
     <AppWrapper>
       <Notifications />
       <FunnelPage
-        accentColorA={InfluxColors.Magenta}
-        accentColorB={InfluxColors.Amethyst}
-        backgroundColor={InfluxColors.DeepPurple}
```

Comment:
Nice.  Does this give us our logo back?

Reply:
Logo's still there, it just changes the background color to match what's on the quartz site